Assets/Scripts/GUI/RecordCameraPathButton.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 The Tilt Brush Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using UnityEngine;
+using UnityEngine.Localization;
+
+namespace TiltBrush
+{
+
+    public class RecordCameraPathButton : OptionButton
+    {
+
+        public GameObject m_CameraRecordUnsupportedPopup;
+        override protected void OnButtonPressed()
+        {
+#if UNITY_ANDROID || true
```

Comment:
Is this condition left over from testing?

Reply:
Ooops. Yep. Fixed.